app-typescript/components/DateTimePicker.tsx

```diff
@@ -0,0 +1,101 @@
+import * as React from 'react';
+import {DatePicker} from '../components/DatePicker';
+import {Spacer} from '@superdesk/common';
+import {defaultTo, padStart} from 'lodash';
```

Comment:
can you use [padStart](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/padStart) from standard library?

Reply:
Yes